js/client-teambuilder.js

```diff
@@ -2149,10 +2155,11 @@
 				}
 			}
 
-			this.ignoreEVLimits = (this.curTeam.format === 'classichackmons');
```

Comment:
This is the original position of the line, except that it doesn't actually work if you do it here.